packages/@okta/vuepress-site/docs/guides/unlock-mobile-app-with-biometrics/store-tokens/android/storerefreshtoken.md

```diff
@@ -1,3 +1,28 @@
-Example Code Note: Assume the developer's sign-in method looks like the example from: https://developer.okta.com/docs/guides/sign-into-mobile-app/ios/sign-in-page/
+First create a encryption manager and set it in the `WebAuthBuilder` like the following:
```

Comment:
First create an encryption manager and set it in the `WebAuthBuilder` like the following: